pages/landing/hipaa/template/pricing/index.html

```diff
@@ -18,31 +18,31 @@
     description:
       Early access subscribers will receive a CIS staging environment when invited<br>
       and a fully HIPAA-compliant production infrastructure when it’s ready.
-    price: 3060
-    strike_price: 10000
-    flat_fee: 7425
-    strike_flat_fee: 9900
+    price: "3,060"
+    strike_price: "10,000"
+    flat_fee: "7,425"
+    strike_flat_fee: "9,900"
```

Comment:
Do we need to charge a flat fee at all? It just seems to complicate the decision and doesn't add that much revenue compared to the monthly, recurring fee.

Reply:
Or perhaps that's one of the big discounts you get as an early subscriber? No "flat fee"?

Reply:
Updated in 83985751.